gapstool.vbs

```diff
@@ -306,14 +368,17 @@ Sub RunDiagnostics(CompName)
   RunCommand "dir ""\\" & CompName & "\c$\Program Files (x86)\Google\Google Apps Password Sync"" /B /S", _
              "instx86"
 
-  PrintLine "Geting system-wide proxy settings dump from registry - proxy.txt"
+  PrintLine "Getting system-wide proxy settings dump from registry - proxy.txt"
   RunCommand "reg query ""\\" & CompName & "\HKLM\SOFTWARE\Policies\Microsoft\Windows\CurrentVersion\Internet Settings"" /v ProxySettingsPerUser", _
              "proxy"
 
   PrintLine "Getting system-wide WinHTTP settings dump from registry - winhttp.txt"
   RunCommand "reg query ""\\" & CompName & "\HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Connections"" /v WinHttpSettings", _
              "winhttp"
 
+  PrintLine "Getting system-wide WinHTTP settings dump from registry, and decoding - winhttp_decoded.txt"
+  DecodeWinHTTPSettings CompName, LogDir & "\winhttp_decoded.txt"

```

Comment:
You should save the file to `winhttp_decoded.txt` instead of the path with `LogDir`, because it's already the current directory (see line 293).